Mathlib/Data/Fintype/Shrink.lean

```diff
@@ -0,0 +1,18 @@
+/-
+Copyright (c) 2024 Yaël Dillies. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies
+-/
+import Mathlib.Data.Countable.Small
+import Mathlib.Data.Fintype.Card
+
+/-!
+# Fintype instance for `Shrink`
+-/
+
+universe u v
+variable {α : Type u} [Fintype α]
+
+noncomputable instance Shrink.instFintype : Fintype (Shrink.{v} α) := .ofEquiv _ (equivShrink _)
+
+@[simp] lemma Fintype.card_shrink : card (Shrink.{v} α) = card α := ofEquiv_card _
```

Comment:
This seems like a case where we should give `[Fintype (Shrink.{v} α)]` explicitly too, for the usual reasons. It's of course easy to adjust the proof.